modules/develop/pages/consume-data/consumer-offsets.adoc

```diff
@@ -4,20 +4,127 @@
+When a consumer group works together to consume data from topics, the partitions are divided among the consumers in the group. For example, if a topic has 12 partitions, and there are two consumers, each consumer would be assigned six partitions to consume. If a new consumer starts later and joins this consumer group, a rebalance occurs, such that each consumer ends up with four partitions to consume. You specify a consumer group by setting the `group.id` property to a unique name for the group.
+
+Kafka tracks the maximum offset it has consumed in each partition and can commit offsets to ensure it can resume processing from the same point in the event of a restart. Kafka allows offsets for a consumer group to be stored on a designated broker, known as the group coordinator. All consumers in the group send their offset commits and fetch requests to this group coordinator.
+
+NOTE: More advanced consumers can read data from Redpanda without using a consumer group, by requesting to read a specific topic + partition + offset range. This pattern is often used by stream processing systems such as Apache Spark and Apache Flink, which have their own mechanisms for assigning work to consumers.
+
+When the group coordinator receives an OffsetCommitRequest, it appends the request to the https://kafka.apache.org/documentation/#compaction[compacted^] Kafka topic `pass:[__consumer_offsets]`.
 The broker sends a successful offset commit response to the consumer only after all the replicas of the offsets topic receive the offsets.
 If the offsets fail to replicate within a configurable timeout, the offset commit fails and the consumer may retry the commit after backing off.
-The brokers periodically compact the offsets topic, because it only needs to maintain the most recent offset commit for each partition.
+The brokers periodically compact the `pass:[__consumer_offsets]` topic, because it only needs to maintain the most recent offset commit for each partition.
 The coordinator also caches the offsets in an in-memory table to serve offset fetches quickly.
 
-When stored in a partition, a sequential number is assigned to each event for order. This number is the _offset_. The offset allows consumer event processing to start or continue from where it left off. So, if you have a consumer application outage that prevents your application from receiving events, you can use the consumer offset to select and send only the events that occurred when the application was down. By default, the offset starts at 0, but you can manually specify a specific start value.
+== Commit strategies
+
+There are several strategies for managing offset commits:
+
+=== Auto commit
+
+Auto commit is the default commit strategy, where the client automatically commits offsets at regular intervals. You can enable this by setting `enable.auto.commit` to `true`. The client will then commit offsets every `auto.commit.interval.ms` milliseconds.
```

Comment:
```suggestion
Auto commit is the default commit strategy, where the client automatically commits offsets at regular intervals. This is set with the `enable.auto.commit` property. The client then commits offsets every `auto.commit.interval.ms` milliseconds.
```
Just my preference, because I still thought it sounded like you had to enable it.